tests/integration/test_wazuh_db/test_wdb_backup_configs.py

```diff
@@ -0,0 +1,229 @@
+from wazuh_testing.tools import LOG_FILE_PATH, WAZUH_PATH
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_db_backups_conf.yaml')
+backups_path = os.path.join(WAZUH_PATH, 'backup', 'db')
+interval = 5
+
+parameters = [{'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':''},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':'value'},
+              {'ENABLED': 'yes', 'INTERVAL': '', 'MAX_FILES':'1'},
+              {'ENABLED': 'yes', 'INTERVAL': 'value', 'MAX_FILES':1},
+              {'ENABLED': 'no', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':3},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':0}
+            ]
+
+metadata = [{'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':''},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':'value'},
+            {'ENABLED': 'yes', 'INTERVAL': '', 'MAX_FILES':1},
+            {'ENABLED': 'yes', 'INTERVAL': 'value', 'MAX_FILES':1},
+            {'ENABLED': 'no', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':3},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':0}
+           ]
+
+configurations = load_wazuh_configurations(configurations_path, __name__ ,
+                                           params=parameters, metadata=metadata)
+
+# Variables
+BACKUP_CREATION_CALLBACK = r'.*Created Global database backup "(backup/db/global.db-backup.*.gz)"'
+WRONG_INTERVAL_CALLBACK = r".*Invalid value for element ('interval':.*)"
+WRONG_MAX_FILES_CALLBACK = r".*Invalid value for element ('max_files':.*)"
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+def restart_wazuh():
+    """Restarts Wazuh."""
+    control_service('stop')
+    control_service('start')
+
+
+def validate_interval_format(interval):
+    if interval=='':
+        return False
+    if interval[-1] not in ['s','m', 'h','d','w','y'] or not isinstance(int(interval[0:-1]), numbers.Number):
+        return False
+    return True
+
+
+# Fixtures
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope='function')
+def clear_logs(get_configuration, request):
+    """Reset the ossec.log and start a new monitor"""
+    truncate_file(LOG_FILE_PATH)
+    file_monitor = FileMonitor(LOG_FILE_PATH)
+    setattr(request.module, 'wazuh_log_monitor', file_monitor)
```

Comment:
Why you don't use the restart_wazuh method from tests\integration\conftest.py?  It contains restart service and removes logs. Also, it's a fixture.